pkg/trace/obfuscate/sql.go

```diff
@@ -265,15 +300,17 @@ func (oq *ObfuscatedQuery) Cost() int64 {
 // attemptObfuscation attempts to obfuscate the SQL query loaded into the tokenizer, using the
 // given set of filters.
 func attemptObfuscation(tokenizer *SQLTokenizer) (*ObfuscatedQuery, error) {
+
 	var (
-		tableFinder    = &tableFinderFilter{}
-		useTableFinder = config.HasFeature("table_names")
-		out            = *bytes.NewBuffer(make([]byte, 0, len(tokenizer.buf)))
-		err            error
-		lastToken      TokenKind
-		discard        discardFilter
-		replace        replaceFilter
-		grouping       groupingFilter
+		storeTableNames = config.HasFeature("table_names")
+		normalizeTables = config.HasFeature("normalize_sql_tables")
+		out             = *bytes.NewBuffer(make([]byte, 0, len(tokenizer.buf)))
+		err             error
+		lastToken       TokenKind
+		discard         discardFilter
+		replace         = &replaceFilter{normalizeTables: normalizeTables}
```

Comment:
I'm not 100% sure, but I think that this might be more efficient if it were not a pointer. Same for the other filters.